global.json

```diff
@@ -1,10 +1,10 @@
 {
   "sdk": {
-    "version": "6.0.403",
+    "version": "8.0.107",
```

Comment:
The github workflow `setup-dotnet` steps can be updated to change the sdk versions being installed.

Reply:
Resolved